Lesson26/Gokhan/index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>Image Gallery</title>
+  <link rel="stylesheet" type="text/css" href="./styles.css">
+</head>
+
+<body>
+
+  <h3>Image Gallery</h3>
+
+  <br>
```

Comment:
Avoid using `<br>` element, use margin (or gap with flexbox and grid) to add spacing.